internal/domain/bonus/service/interface.go

```diff
@@ -0,0 +1,7 @@
+package service
+
+import http_gw "github.com/Azzonya/gophermart/internal/client/accrual/http"
+
+type RepoAccrualI interface {
```

Comment:
```suggestion
type ClientAccrual interface {
```
он не репозиторий все таки и клиентский код должен это понимать